src/game/room.c

```diff
@@ -736,11 +739,41 @@ void Room_PopulateSectorData(
 
 void Room_TestTriggers(const ITEM_INFO *const item)
 {
-    const bool is_heavy = item->object_id != O_LARA;
     int16_t room_num = item->room_num;
-    const SECTOR_INFO *const sector =
+    const SECTOR_INFO *sector =
         Room_GetSector(item->pos.x, MAX_HEIGHT, item->pos.z, &room_num);
 
+    Room_TestSectorTrigger(item, sector);
+    if (item->object_id == O_LARA || !g_GameFlow.use_extended_triggers) {
```

Comment:
Thanks. Is the torso boss the only enemy for which this criterion is true? If so, we could just hardcode it to always trigger this behavior for it only, and skip the gameflow changes.

Reply:
Yes, good point. That'll be cleaner, I'll sort that out.